packages/aws-cdk/bin/cdk.ts

```diff
@@ -113,7 +113,8 @@ async function parseCommandLineArguments() {
       .option('exclusively', { type: 'boolean', alias: 'e', desc: 'Only diff requested stacks, don\'t include dependencies' })
       .option('context-lines', { type: 'number', desc: 'Number of context lines to include in arbitrary JSON diff rendering', default: 3, requiresArg: true })
       .option('template', { type: 'string', desc: 'The path to the CloudFormation template to compare with', requiresArg: true })
-      .option('strict', { type: 'boolean', desc: 'Do not filter out AWS::CDK::Metadata resources', default: false }))
+      .option('strict', { type: 'boolean', desc: 'Do not filter out AWS::CDK::Metadata resources', default: false })
+      .option('security-check-only', { type: 'boolean', desc: 'Only diff for broadened security changes', default: false }))
```

Comment:
Can we change this to:

* `--security-only`

Instead?

Or do you suppose it would read better to do:

* `--only-security` ?